src/server/models/db.js

```diff
@@ -9,7 +9,11 @@ module.exports.connect = async function (url) {
     return;
```

Comment:
@jackvanb This appears to be missing require('dotenv').config(); from here:

https://github.com/ucla/lti-media-resources/pull/238/files#diff-1497d44a565c9a4d1e06aaa06e751e9cd1c3e7b47ffa1d7829a6ae4e6405a11cR1